app/view/twig/editcontent/fields/_relationship.twig

```diff
@@ -6,10 +6,12 @@
 {# Use option groups #}
 {% set groupby = relation.groupby|default(false) %}
 
+{% set values = context.content.relation.getField(relcontenttype, true, context.contenttype.slug, context.content.id)|default([]) %}
+
 {# Currently selected relationship values #}
 {% set relselect = [] %}
-{% for item in context.content.relation.getField(relcontenttype, true)|default([]) %}
-    {% set relselect =  relselect|merge([item.to_id]) %}
+{% for item in values %}
+    {% set relselect =  relselect|merge([item.getToId()]) %}
 {% endfor %}
```

Comment:
Well, the problem is that since this can be an inversed relation that will get the actual to_id, instead of the inversed to_id if i use twigs getters. see https://github.com/bolt/bolt/blob/858d652e49c3b61fef7e0f66f47750a4fe97d766/src/Storage/Entity/Relations.php#L81-L88